examples/auth/src/getPathname.ts

```diff
@@ -0,0 +1,18 @@
+import { Location } from "react-router"
+
+const isObjectWithKey = <T extends string>(
```

Comment:
Using something like Lodash `get` could replace this but I didn't want to rely on a dependency.